skema/text_reading/text_reading/grounding/src/test/scala/org/ml4ai/grounding/TestMiraEmbeddingsGrounder.scala

```diff
@@ -1,45 +1,48 @@
 package org.ml4ai.grounding
 
+import breeze.linalg._
 import com.typesafe.config.ConfigFactory
 import org.clulab.utils.FileUtils
+import org.json4s.JsonDSL.boolean2jvalue
 import org.ml4ai.skema.common.test.Test
-
 import org.scalatest.OptionValues._
+import breeze.stats.distributions._
 
 class TestMiraEmbeddingsGrounder extends Test {
 
   // Lazily load the grounder. We assume it's state and behavior is immutable
   // So we can build it once and reuse it as necessary in the suite
+
   val miraEmbeddingsGrounder: MiraEmbeddingsGrounder = {
     val config = ConfigFactory.load().getConfig("Grounding")
     val ontologyPath = config.getString("ontologyPath")
     // val embeddingsPath = config.getString("embeddingsPath")
 
-    MiraEmbeddingsGrounder(ontologyPath, None)
+    MiraEmbeddingsGrounder(ontologyPath, None, 10, 0.25.toFloat)
```

Comment:
0.25f

Reply:
change for 0.25.toFloat to 0.25f done! https://github.com/ml4ai/skema/commit/7a49c0501223457b22bb01eec10ebca46821b2f0